cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,35 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
-  });
-
-  it('', () => {
+describe('Conduit', () => {
+  const userName = 'nevermore';
+  const email = 'nevermore@gmail.com';
+  const password = 'nevermore123';
+  const title = 'title';
+  const description = 'description';
+  const body = 'body';
 
+  beforeEach(() => {
+    cy.visit('/');
+    cy.login(email, userName, password);
   });
 
-  it('', () => {
+  it('should create article with valid data', () => {
+    cy.reload();
+    cy.contains('a', 'New Article').click();
+    cy.get('[placeholder="Article Title"]').type(title);
+    cy.get('[placeholder="What\'s this article about?"]').type(description);
+    cy.get('[placeholder="Write your article (in markdown)"]').type(body);
+    cy.get('.btn-primary').click();
+    cy.get('.container').should('contain', title)
+      .and('contain', 'Delete Article')
+      .and('contain', 'Edit Article');
+  });
```

Comment:
You are creating an article in the 'should create article with valid data' test but not deleting it afterwards. This can lead to data pollution if you run your tests multiple times. Consider cleaning up (deleting the article) at the end of the test.